python/tank/bootstrap/manager.py

```diff
@@ -221,11 +244,50 @@ def set_progress_callback(self, callback):
         """
         self._progress_cb = callback
 
-    def bootstrap_engine(self, engine_name, entity=None):
+    def bootstrap_engine(self,
```

Comment:
My goal while implementing _threading.py_ was to keep the Qt dependencies confined in there while using QThread, signals ans slots (as outlined in your original suggestion).